{bio}[foss/2023b] BWA v0.7.17

grpiccoli opened the pull request:
(created using `eb --new-pr`)

verdurin commented on easybuild/easyconfigs/b/BWA/BWA-0.7.17-foss-2023b.eb:
```diff
@@ -0,0 +1,32 @@
+##
+name = 'BWA'
+version = '0.7.17'
+homepage = 'http://bio-bwa.sourceforge.net/'
```
```suggestion
homepage = 'https://bio-bwa.sourceforge.net/'
```
Looks like `https` works now, at last.

verdurin commented on easybuild/easyconfigs/b/BWA/BWA-0.7.17-foss-2023b.eb:
```diff
@@ -0,0 +1,32 @@
+homepage = 'http://bio-bwa.sourceforge.net/'
+description = """Burrows-Wheeler Aligner (BWA) is an efficient program that aligns
+ relatively short nucleotide sequences against a long reference sequence such as the human genome."""
+toolchain = {'name': 'foss', 'version': '2023b'}
```
I think some of the blank lines between sub-sections have been lost, which makes it less readable.

Please add a blank line here before `toolchain`, and another after `toolchainopts`.

verdurin commented:
@grpiccoli Welcome to EasyBuild, and thanks for your contribution.
I've added a couple of comments, with suggestions for small fixes.

grpiccoli commented:
Hi @verdurin, thank you for your welcome and feedback.
I just pushed a corrected version using --update-pr.
Cheers

grpiccoli reviewed (commented):
Thank you for your feedback

grpiccoli reviewed (commented):
approved

grpiccoli commented:
Hi @verdurin,

I deleted the original file which was using foss 2023b and switch it for foss 2023a, so you will not see your changes reflected on the file that was deleted but the new file instead. I also added you as a contributor,

Thanks

grpiccoli commented:
Closing this PR and opening a fresh one, Cheers
